### ISSUE TITLE
Filters of FileDialog not working

### ISSUE THREAD

**Issue by PTKDrake** [yue/yue#145]
Yue Version: gui@0.11.0
Operating System: Windows 11

Code:
```ts
let fp = '';
    const dialog = gui.FileOpenDialog.create();
    dialog.setTitle('Select Symbol File');
    dialog.setFilters([
        {description: 'Symbol File', extensions: ['jpg', 'png']}
    ]);
    if (dialog.runForWindow(win)) {
        fp = dialog.getResult();
        console.log(fp);
    }
```
I tried changing the extension to *.png or .png neither worked.

![image](https://user-images.githubusercontent.com/24309803/178103999-cf2747a6-f9c3-41b4-8609-c3abd7e7c512.png)

**Comment by PTKDrake** [comment]
Can anyone help me please?

**Comment by zcbenz** [comment]
Fixed with 7c3346b.
